docker-compose.yaml

```diff
@@ -33,6 +33,30 @@ services:
       labels:
         - "prometheus-job=save-orchestrator"
     logging: *loki-logging-jvm
+  sandbox:
+    image: ghcr.io/saveourtool/save-sandbox:${SANDBOX_TAG}
+    user: root  # to access host's docker socket
+    environment:
+      - "SPRING_PROFILES_ACTIVE=${PROFILE}"
+      - "MYSQL_USER=/run/secrets/db_username"
+      - "MYSQL_PASSWORD_FILE=/run/secrets/db_password"
+    secrets:
+      - db_username
+      - db_password
+      - db_url
```

Comment:
I believe our `db_url` already contains schema name

Reply:
Maybe we can move URL from secrets into properties because it's not required to have it hidden, I think we've discussed it before

Reply:
done